crypto/src/main/java/org/web3j/crypto/Logger.java

```diff
@@ -0,0 +1,19 @@
+package org.web3j.crypto;
+
+public interface Logger {
```

Comment:
Is there any way we can use slf4j still? This is the one part of the PR I'm not keen on.